scripts/AE2.zs

```diff
@@ -1077,47 +1083,73 @@ recipes.addShaped(UStorageHousing, [
 [CertusScrew, AlPlate, HHammer]]);
 
 // --- Storage Cell Component - 1K
-recipes.addShaped(StorageComponent1K, [
-[PrimitiveCircuit, <dreamcraft:item.ChargedCertusQuartzDust>, PrimitiveCircuit],
-[<dreamcraft:item.ChargedCertusQuartzDust>, <dreamcraft:item.LogicProcessorItemGoldCore>, <dreamcraft:item.ChargedCertusQuartzDust>],
-[PrimitiveCircuit, <dreamcraft:item.ChargedCertusQuartzDust>, PrimitiveCircuit]]);
+//recipes.addShaped(StorageComponent1K, [
+//[PrimitiveCircuit, <dreamcraft:item.ChargedCertusQuartzDust>, PrimitiveCircuit],
+//[<dreamcraft:item.ChargedCertusQuartzDust>, <dreamcraft:item.LogicProcessorItemGoldCore>, <dreamcraft:item.ChargedCertusQuartzDust>],
+//[PrimitiveCircuit, <dreamcraft:item.ChargedCertusQuartzDust>, PrimitiveCircuit]]);
+
+Assembler.addRecipe(StorageComponent1K, [RAMchip*4, <ore:dustChargedCertusQuartz>*4, <dreamcraft:item.LogicProcessorItemGoldCore>,<gregtech:gt.integrated_circuit:1>*0], null, 120, 480);
```

Comment:
why ram chip instead of PrimitiveCircuit?

Reply:
https://github.com/GTNewHorizons/NewHorizons/pull/5040/commits/a34bc5a555d4f61156679c78a213f9e114a19c06